app/app.js

```diff
@@ -2,14 +2,25 @@ import express from 'express';
 import config from './config';
 import { router } from './routes/app.route';
 import { ErrorHandler } from './middlewars/ErrorHandler';
+import { logErrorHandler, logInfoHandler } from './middlewars/logger';
 
 const app = express();
 const PORT = config.get('port');
 
+Promise.reject(new Error('Promise Error'));
+
+process.on('unhandledRejection', err => {
+    logErrorHandler(err);
+});
+
 app.use(express.json({ extended: true }));
 app.use('/', router);
+app.use((req, res, next) => {
+    logInfoHandler(req);
```

Comment:
you can also make it as a middleware: `app.use(loggerMiddleware)`. 
Also, it's good to log info that response was sent, you can use `res.on('finish', ...)` event or `morgan` package

Reply:
Done